src/main/paradox/setup/resources/index-domains.properties

```diff
@@ -0,0 +1,11 @@
+com.bigdata.rdf.sail.namespace=kg-domains
+com.bigdata.rdf.sail.isolatableIndices=false
+com.bigdata.rdf.store.AbstractTripleStore.textIndex=true
+com.bigdata.rdf.store.AbstractTripleStore.quads=true
+com.bigdata.rdf.sail.truthMaintenance=false
+com.bigdata.namespace.test-ns.lex.com.bigdata.btree.BTree.branchingFactor=400
+com.bigdata.rdf.store.AbstractTripleStore.justify=false
+com.bigdata.rdf.store.AbstractTripleStore.geoSpatial=false
+com.bigdata.namespace.test-ns.spo.com.bigdata.btree.BTree.branchingFactor=1024
```

Comment:
test-ns ? is that something we want to put out int he default config ?

Reply:
For now, I guess these namespace configs need to be (manually) applied  prior to start Nexus KG service in order to benefit from indexing and search through blazegraph. 

But the right approach would be to read and applied (user provided or default) namespace configs automatically at boot time.

Reply:
Very nice catch; I've just used the index definitions that were present in the docs. I'll regenerate the indices.

Reply:
Corrected the issue.